Phase01/Phase01-StudentAverage/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Resources;
+
+namespace Phase01_StudentAverage;
+
+internal class Program
+{
+    static void Main()
+    {
+        try
+        {
+            string studentsFilePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, Resources.StudentsFilePath);
+            string coursesFilePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, Resources.CoursesFilePath);
+
+            var topStudentsCalculator = new TopStudentAverageScoreCalculator(studentsFilePath, coursesFilePath);
+            topStudentsCalculator.PrintTopNStudentAverageScore(3);
+        }
+        catch (Exception e)
```

Comment:
catch more specific exceptions (why?)

Reply:
In general, it is better to catch specific exceptions because obviously for each exception, the is a different exception handling we want to do.
but the reason which may effect us in this example, is that using a catch-all exception can unintentionally swallow exceptions that you did not anticipate, which make the debugging pretty hard.